core/src/main/resources/lib/layout/breadcrumbBar.jelly

```diff
@@ -34,42 +34,38 @@ THE SOFTWARE.
     use the &lt;l:breadcrumb> tag.
   </st:documentation>
 
-  <tr id="top-nav">
-    <td id="left-top-nav" colspan="2">
-      <st:adjunct includes="lib.layout.breadcrumbs" />
-      <div class="top-sticker noedge">
-        <div class="top-sticker-inner">
-          <div class="breadcrumbs__wrapper">
-            <ul id="breadcrumbs">
-              <j:forEach var="anc" items="${request.ancestors}">
-                <j:if test="${h.isModel(anc.object) and anc.prev.url!=anc.url}">
-                  <li class="item">
-                    <a href="${anc.url}/" class="${h.isModelWithContextMenu(anc.object)?'model-link':null} inside">
-                      ${anc.object.displayName}
-                    </a>
+  <st:adjunct includes="lib.layout.breadcrumbs" />
+  <div class="top-sticker noedge">
+    <div class="top-sticker-inner">
+      <div class="breadcrumbs__wrapper">
+        <ul id="breadcrumbs" role="menubar">
+          <j:forEach var="anc" items="${request.ancestors}">
+            <j:if test="${h.isModel(anc.object) and anc.prev.url!=anc.url}">
+              <li class="item" role="menuitem" aria-haspopup="true" aria-expanded="false">
```

Comment:
`haspopup` and `expanded` should not be used if we will not then update their values based on the dropdown state. They should be removed IMO

Reply:
Yeah, I think I left this here accidentally as I was hoping to dynamically adjust those properties in the corresponding JS, but I don't think I was able to figure out a clean way to introduce that.

Reply:
Then I suggest removing these attributes